packages/account/src/aliases.ts

```diff
@@ -0,0 +1,7 @@
+import { BiconomySmartAccountV2 } from "./BiconomySmartAccountV2";
+import { BiconomySmartAccountV2Config } from "./utils/Types";
+
+export { BiconomyPaymaster as Paymaster, IPaymaster } from "@biconomy/paymaster";
+export { Bundler, IBundler } from "@biconomy/bundler";
+export const createSmartWalletClient = BiconomySmartAccountV2.create;
+export type SmartWalletConfig = BiconomySmartAccountV2Config;
```

Comment:
> export const createSmartWalletClient = BiconomySmartAccountV2.create;

this would be very helpful.

Also would be better to do this in index.ts rather creating a new file, just to have a minimal codebase.